x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/fetch_graph.ts

```diff
@@ -289,7 +289,7 @@ ${
     ""
   )
 | STATS badge = COUNT(*),
-  uniqueEventsCount = COUNT_DISTINCT(CASE(isAlert == false, event.id, null)),
+  totalEventsCount = COUNT_DISTINCT(event.id),
```

Comment:
As for counting alerts - I'd count by the alert id/doc id - because you might have more than one alert per event.  

As for counting event.id. I'm not sure if it is correct to count both alerts and events. Can you explain the use case you were solving here?